README.md

```diff
@@ -46,74 +41,70 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 <!-- Replace this heading with the name of the root level module (the repo name) -->
 ## terraform-ibm-watsonx-data
 
-### Usage
+This module supports provisioning the following:
 
-<!--
-Add an example of the use of the module in the following code block.
+* Provisioning watsonx.data instance with a selectable service plan
 
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
+### Usage
 
 ```hcl
+module "watsonx_data" {
+    source = "terraform-ibm-modules/watsonx-data/ibm"
+    version = "X.Y.Z" # Replace "X.X.X" with a release version to lock into a specific release
+    resource
+    watsonx_data_name = "watsonx-data"
+    service = "lakehouse"
+    region = "us-south"
+    plan = "lite"
+    resource_group_id     = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
+}
```

Comment:
can you fix the indentation so all the equal signs line up - just like our terraform fmt linter would do